src/Orleans.Core/Logging/ILoggerExtensions.cs

```diff
@@ -95,7 +95,7 @@ public static void Debug(this ILogger logger, int logCode, string format, params
             logger.LogDebug(logCode, format, args);
         }
 
-        internal static void Debug(this ILogger logger, ErrorCode logCode, string format, params object[] args)
```

Comment:
If we organize error codes as described in other comment, we can keep these internal.